frontend/views/pages/Contributions.vue

```diff
@@ -241,17 +241,21 @@ export default {
     openModal (modal) {
       sbp('okTurtles.events/emit', OPEN_MODAL, modal)
     },
-    async handleNonMonetary (type, value) {
+    async handleNonMonetary (type, value, done) {
       try {
         const groupProfileUpdate = await sbp('gi.contracts/group/groupProfileUpdate/create',
           { [type]: value },
           this.$store.state.currentGroupId
         )
+        console.log('send msg!', groupProfileUpdate)
         await sbp('backend/publishLogEntry', groupProfileUpdate)
+        // TODO/BUG - Discover why "value" gets added to the list
+        // but Contribution input takes longer to unmount. (in Slow 3G).
       } catch (e) {
         console.error('handleNonMonetary', e)
         alert(e.message)
       }
+      done()
```

Comment:
- Why was this added? Is it truly necessary? This is already an asynchronous method. Callbacks are usually not used with them.
- If for some reason a callback is really needed, then it's usually called with the error object when there's an error. However, I think this should be removed, and I'm not seeing where it's used

Reply:
As explained on the other comment, this is needed to let the child (`Contribution.vue`) know when to reset the `isSubmitting` back to false.